web/app/models.py

```diff
@@ -226,7 +227,7 @@ def __str__(self):
 class TournamentPlayer(db.Model):
     id = db.Column(db.Integer, primary_key=True, autoincrement=True)
     name = db.Column(db.String(80))
-    player_id = db.Column(db.String(5))
+    aga_num = db.Column(db.String(5))
     member_ex_date = db.Column(db.String(20))
     rating = db.Column(db.String(5))
```

Comment:
Yes, the AGA numbers were originally assumed to be between 0 and 99999. with the last 10k-1 for temporary numbers. But, let's not build that assumption into the data model since I hope that, at least in the future, there are more than 100,000 go players in the AGA database.